packages/core/src/global/js/accessibility.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-inner-declarations */
+
+function setupAccessibility() {
+  if (typeof document !== 'undefined') {
+    document.addEventListener('mousedown', removeAccessibilityClass);
```

Comment:
this might warrant more research, but i'm not sure that using a mouse negates the need for accessibility styles. users might use a mouse but still require clear accessible focus states.

Reply:
I analyzed some pretty high profile sites and this was a very common pattern I found

Reply:
We should do some future investigation into pointer event support.